app/templates/profile.hbs

```diff
@@ -2,23 +2,114 @@
-        <li>{{link-to 'Census' 'profile.census'}}</li>
+        <li>
+          {{#if (eq tab 'profile.census')}}
+            {{#tab-dropdown
+              tabName='Census'
+              tabNameSmall=''
+              as |context| }}
+              <li>{{#scroll-to href='#' afterScroll=context.closeTabDropdown}}Section{{/scroll-to}}</li>
+            {{/tab-dropdown}}
+          {{else}}
+            {{link-to 'Census' 'profile.census'}}
+          {{/if}}
+        </li>
         {{#if (eq summaryLevel 'blocks')}}
           <li><a class="disabled">Demographic <small>(ACS)</small> <sup class="warning">{{fa-icon 'warning'}}{{tooltip-on-element text="ACS data not available at block level."}}</sup></a></li>
           <li><a class="disabled">Social <small>(ACS)</small> <sup class="warning">{{fa-icon 'warning'}}{{tooltip-on-element text="ACS data not available at block level."}}</sup></a></li>
           <li><a class="disabled">Economic <small>(ACS)</small> <sup class="warning">{{fa-icon 'warning'}}{{tooltip-on-element text="ACS data not available at block level."}}</sup></a></li>
           <li><a class="disabled">Housing <small>(ACS)</small> <sup class="warning">{{fa-icon 'warning'}}{{tooltip-on-element text="ACS data not available at block level."}}</sup></a></li>
         {{else}}
-          <li>{{#link-to 'profile.demographic'}}Demographic  <small>(ACS)</small>{{/link-to}}</li>
-          <li>{{#link-to 'profile.social'}}Social  <small>(ACS)</small>{{/link-to}}</li>
-          <li>{{#link-to 'profile.economic'}}Economic  <small>(ACS)</small>{{/link-to}}</li>
-          <li>{{#link-to 'profile.housing'}}Housing  <small>(ACS)</small>{{/link-to}}</li>
+          <li>
+            {{#if (eq tab 'profile.demographic')}}
+              {{#tab-dropdown
+                tabName='Demographic'
+                tabNameSmall='(ACS)'
+                as |context| }}
+                <li>{{#scroll-to href='#age-and-sex' afterScroll=context.closeTabDropdown}}Age and Sex{{/scroll-to}}</li>
+                <li>{{#scroll-to href='#mutually-exclusive-race-hispanic-origin' afterScroll=context.closeTabDropdown}}Mutually Exclusive Race / Hispanic Origin{{/scroll-to}}</li>
+                <li>{{#scroll-to href='#hispanic-subgroup' afterScroll=context.closeTabDropdown}}Hispanic Subgroup{{/scroll-to}}</li>
+                <li>{{#scroll-to href='#asian-subgroup' afterScroll=context.closeTabDropdown}}Asian Subgroup{{/scroll-to}}</li>
+              {{/tab-dropdown}}
+            {{else}}
+              {{#link-to 'profile.demographic'}}Demographic <small>(ACS)</small>{{/link-to}}
+            {{/if}}
+          </li>
+          <li>
+            {{#if (eq tab 'profile.social')}}
+              {{#tab-dropdown
+                tabName='Social'
+                tabNameSmall='(ACS)'
+                as |context| }}
+                <li>{{#scroll-to href='#household-type' afterScroll=context.closeTabDropdown}}Household Type{{/scroll-to}}</li>
```

Comment:
Looks great - this is non-blocking, just a comment, but we could go even further and have the `tab-dropdown` component yield another contextual component like this:

```handlebars
// tab-dropdown.hbs
// ... other stuff above

{{yield (hash scroll-to=(component 'scroll-to' afterScroll=(action 'closeTabDropdown')))}}
```

Then in our profile view, we could say:
```handlebars
// profile.hbs
{{#tab-dropdown 
      tabName='Demographic'
      tabNameSmall='(ACS)'
                as |dropdown| }}
       <li>{{#dropdown.scroll-to href='#household-type'}}Household Type{{/dropdown.scroll-to}}</li>
        // more
{{/tab-dropdown}}
```

Thoughts? Want to try this out or move on?